### DISCUSSION TITLE
secretflow添加组件，源码构建时参数过长的问题

### DISCUSSION THREAD

**Discussion by kongshijie** [secretflow/secretflow#1581] [Q&A]
Argument list too long
 
![微信图片_20241113095240](https://github.com/user-attachments/assets/9e241411-854f-46c5-8208-21adef55772a)

 82 #Building multi platform images
 83 docker buildx build \
 84   --platform linux/arm64,linux/amd64 \
 85   -f anolis.Dockerfile \
 86   -t ${IMAGE_TAG} \
 87   --build-arg sf_version=${VERSION} \
 88   --build-arg config_templates="$(cat config_templates.yml)" \
 89   --build-arg deploy_templates="$(cat deploy_templates.yml)" \
 90   --build-arg comp_list="$(cat comp_list.json)" \
 91   --build-arg translation="$(cat translation.json)" \
 92   . --push

**Comment by lanyy9527** [comment]
暂时需要将生成的comp_list.json和translation.json文件中的内容，手动写入到Dockerfile中，需要注意对引号、回车符、反斜杠进行转译处理；该问题在后续的版本中会进行修复。
